cocotb/scoreboard.py

```diff
@@ -97,7 +97,9 @@ def compare(self, got, exp, log, strict_type=True):
             self.errors += 1
             log.error("Received transaction is a different type to expected "
                       "transaction")
-            log.info("Got: %s but expected %s" %
+            log.info("Got: %s but expected %s "
```

Comment:
Received seems much more polite than Got as well.

Reply:
Some standardization on the way such things are printed would be nice across all of cocotb.
Personally I have used some easily parseable key=value format: "(actual=..., expected=...)" or even just "(act=..., exp=...)".

Reply:
I can make the change there, I was just keeping the original message untouched as much as possible.

Reply:
@stuarthodgson @cmarqu Please see 
5480a78